modules/javafx.graphics/src/main/java/javafx/scene/SpotLight.java

```diff
@@ -189,7 +183,7 @@ public final DoubleProperty outerAngleProperty() {
      * than the inner angle but less than the outer angle receive partial intensity governed by this factor. The larger
      * the falloff, the sharper the drop in intensity from the inner cone. A falloff factor of 1 gives a linear drop in
      * intensity, values greater than 1 give a convex drop, and values smaller than 1 give a concave drop. Negative
-     * values are allowed, but give unrealistic lighting.
```

Comment:
I would also add the same language about the "valid range" being `falloff >= 0`.